x/programs/cmd/simulator/src/lib.rs

```diff
@@ -91,6 +91,49 @@ pub enum Key {
     Secp256r1(String),
 }
 
+#[derive(Clone, Debug, PartialEq, Default, Serialize)]
+#[serde(rename_all = "camelCase")]
+#[non_exhaustive]
+pub struct TestContext {
+    program_id: Id,
+    actor_key: Option<Key>,
+    height: u64,
+    timestamp: u64,
```

Comment:
Why not make all of these fields `pub`, this is only meant to be used in tests, and there are no properties that you need to be private